src/middlewared/middlewared/alembic/versions/24.04/2023-09-07_19-50_add_exporting_table_for_reporting.py

```diff
@@ -49,7 +49,7 @@ def upgrade():
             'destination_ip': graphite_ip[0],
             'destination_port': 2003,
             'prefix': 'scale',
-            'hostname': hostname,
+            'namespace': hostname,
```

Comment:
Why are we editing an existing migration?

Reply:
Thank you @anodos325 for catching that - was a leftover of my debugging..